third_party/proton/proton/__init__.py

```diff
@@ -9,3 +9,4 @@
     profile,
     DEFAULT_PROFILE_NAME,
 )
+from .language import record
```

Comment:
Can we take out this line?

Otherwise, `proton.record` seems like recording something **on the CPU**.

The assumption I had was that everything in the `language` package will be JIT compiled.

Reply:
Updated